packages/x-tree-view-pro/src/RichTreeViewPro/RichTreeViewPro.tsx

```diff
@@ -63,6 +64,32 @@ function WrappedTreeItem<R extends {}>({
   return <Item {...itemProps}>{children}</Item>;
 }
 
+WrappedTreeItem.propTypes = {
```

Comment:
Nitpick: Maybe this one should/could be marked with `@ignore - do not document.`? 🤔

Reply:
The virtualization PR totally removes this component so I think we can keep it as is for now :+1: